gtfs/__main__.py

```diff
@@ -67,109 +86,151 @@ def list_feeds(
         ),
     ] = False,
 ) -> None:
-    """Filter feeds spatially based on bounding box."""
-    if bbox is None and predicate is not None:
-        raise typer.BadParameter(
-            f"Please pass a bbox if you want to filter feeds spatially based on predicate = {predicate}!"
-        )
-    elif bbox is not None and predicate is None:
-        raise typer.BadParameter(
-            f"Please pass a predicate if you want to filter feeds spatially based on bbox = {bbox}!"
-        )
+    """Filter feeds spatially based on bounding box or search string.
+
+    :param bbox: set of coordinates to filter feeds spatially
+    :param predicate: the gtfs feed should intersect or should be contained inside the user's bbox
+    :param search: Search for feeds based on a string.
+    :param pretty: display feeds inside a pretty table
+    """
+    sources: list = feed_sources
+
+    if search is not None:
+        if bbox is not None or predicate is not None:
+            raise typer.BadParameter("Please pass either bbox or search, not both at the same time!")
+        else:
+            sources = [
+                src
+                for src in feed_sources
+                if search.lower() in src.__name__.lower() or search.lower() in src.url.lower()
```

Comment:
I'd just look in the name, the URL could contain anything

Reply:
Yeah right